.babelrc

```diff
@@ -1,3 +1,10 @@
 {
-  "presets": ["es2015", "stage-0", "react"]
+  "presets": [
+    ["es2015", {"modules": false}],
```

Comment:
@dlebedynskyi agreed, apparently there's a deprecation message on -latest that says as much now.